lib/config/themes.dart

```diff
@@ -267,7 +268,9 @@ abstract class TwakeThemes {
         iconTheme: IconThemeData(
           size: iconSize,
           color: brightness == Brightness.light
-              ? LinagoraSysColors.material().onBackground
+              ? isColumnMode(context)
```

Comment:
be careful on changing in this file, because it may effect the whole color of the app

Reply:
looks like it effects the color of bottom color 
![IMG_A81DC35FF18F-1](https://github.com/user-attachments/assets/36b66734-8d5e-4a0f-b030-50a0922f1958)

Reply:
fixed